coderd/rbac/roles.go

```diff
@@ -121,7 +121,7 @@ func ReloadBuiltinRoles(opts *RoleOptions) {
 		opts = &RoleOptions{}
 	}
 
-	var ownerAndAdminExceptions []Object
+	ownerAndAdminExceptions := []Object{ResourceWorkspaceLocked}
```

Comment:
We might want to keep `read` access here. Should users be able to read a locked workspace? So they can see it is locked?

Reply:
Yup absolutely, def a bug

Reply:
I was wrong, this only affects WorkspaceBuild. Not Workspace. So this is correct.